### PR TITLE
Add Python 3.9 to testing

### PR DESCRIPTION
**Pull request by judahrand**
#### Motivation

Pretty self explanatory. Should probably test on the newest Python version.

### CODE REVIEW

File: .github/workflows/python-package-win.yml [hunk 1]
```diff
@@ -21,6 +21,9 @@ jobs:
           - python-version: '3.8'
             toxenv: "py38-doctest"
 
+          - python-version: '3.9'
+            toxenv: "py38-doctest"
```
Comment (piskvorky) [inline .github/workflows/python-package-win.yml:25]: Why py38?

**Review by piskvorky — CHANGES_REQUESTED** [review]

File: .github/workflows/python-package-win.yml (hunk 1 above)
Comment (judahrand) [reply .github/workflows/python-package-win.yml:25]: Whoops! Late night typo.

Comment (judahrand) [reply .github/workflows/python-package-win.yml]: Sorted now!

**Comment by piskvorky** [comment]
Thanks! Let's wait for @mpenkov 's blessing :)

**Review by mpenkov — COMMENTED** [review]
Looks great! Thank you @judahrand !

**Pull request merged**